tensorbay/cli/utility.py

```diff
@@ -307,3 +307,49 @@ def shorten(origin: str) -> str:
 
     """
     return origin[:7]
+
+
+def _generate_hint(hint_message: Tuple[str, ...]) -> str:
+    """Generate complete hint message.
+
+    Arguments:
+        hint_message: The information of the draft.
+
+    Returns:
+        The complete hint message.
+
+    """
+    if len(hint_message) == 1:
+        return hint_message[0]
+
+    original_hint, title, description = hint_message
+    hint = [title]
+    if description:
+        hint.extend(["", description])
+    hint.append(original_hint)
+    return "\n".join(hint)
+
+
+def edit_draft_info(
+    message: Tuple[str, ...], error_message: str, hint_message: Tuple[str, ...]
+) -> Tuple[str, str]:
+    """Edit draft information.
+
+    Arguments:
+        message: The message of the draft.
+        error_message: Error message when an error occurs.
+        hint_message: The information of the draft.
+
+    Returns:
+        The extracted title and the description.
+
+    """
+    if message:
+        title, description = message[0], "\n".join(message[1:])
+    else:
+        hint = _generate_hint(hint_message)
```

Comment:
Maybe this can be moved outside `edit_draft_info` to first create the hint and then pass it here.